### DISCUSSION TITLE
Hello! There was a question about the advisability of installing on my laptop

### DISCUSSION THREAD

**Discussion by lev666** [sebanc/brunch#1519] [Q&A]
I am looking for a system for watching YouTube and gatherings in the browser for a long time (preferably 3-5 hours)

But Windows 11 is not enough for me, the laptop keeps a maximum of 2 hours in it. Will I be able to achieve the desired result in Chrome OS?

Notebook specifications: ryzen 7 5800h
rtx 3060 + amd graphics vega 8 (5000)
Laptop model - acer an515-45-r6cu

**Comment by tejasraman** [comment]
you will :)
Use the zork image and it will all go fine(probably)
Chrome OS is lightweight and gives a long battery life, but I'm not sure if your battery is the problem. ChromeOS will make the battery life longer though :)

You need a USB stick to replace Windows entirely, so a heads up

**Reply by lev666** [reply]
> 

It’s just that with the help of brunch, I already installed Chromeos 97 for a long time. But for some reason, the duration of the work was two times worse than on Windows. So I thought, maybe now the situation has become better with the duration of work, or not?

**Comment by tejasraman** [comment]
That RTX 3060 is your problem - it is draining power and this is a common issue with gaming laptops(that is what you have)

**Reply by lev666** [reply]
> 

Is it possible to turn it off somehow?  Through the BIOS it will not work, I already tried it.  Maybe there is some kernel command like on "hacintosh -wegnoegpu"

**Comment by tejasraman** [comment]
I wish… although opencore makes this easy(at least in my experience) there is no way to modify acpi using Linux. I am really not sure how to disable an RTX, but I know that it wil, drain battery and is very much unsupported. You could try bumblebee?

**Reply by lev666** [reply]
> I wish… although opencore makes this easy(at least in my experience) there is no way to modify acpi u see Linux. I am really not sure how to disable an RTX, but I know that it wil, drain battery and is very much unsupported. You could try bumblebee?
> […](#)
> On Apr 16, 2022, 10:57 AM -0600, lev666 ***@***.***>, wrote: Is it possible to turn it off somehow? Through the BIOS it will not work, I already tried it. Maybe there is some kernel command like on "hacintosh -wegnoegpu" — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

What is "bumblebee"?

**Reply by tejasraman** [reply]
In #357, @sebanc had a good solution - adding this as a kernel parameter to solve battery life problems.

"module_blacklist=nouveau" will disable the NVidia driver, putting it to sleep and enabling the built in Vega8 to drive the display.

See #357 for more info

Bumblebee is a Linux way to do it :)

**Reply by tejasraman** [reply]
See the comment that @lupohirp made in #357 about this on his Dell laptop with an NVidia gpu.

**Reply by tejasraman** [reply]
I can provide a video on doing so if needed

**Reply by lev666** [reply]
> В #357 ,@sebancбыло хорошее решение - добавление этого в качестве ядра для решения проблем со временем автономной работы.
> 
> «module_blacklist=nouveau» отключает драйвер NVidia, переведет его в спящий режим и позволит управлять дисплеями встроенной Vega8.
> 
> См. [№ 357](https://github.com/sebanc/brunch/issues/357) для получения дополнительной информации.
> 
> Bumblebee — это Linux-способ сделать это :)

Did I post in the right place?
![IMG_20220418_073651](https://user-images.githubusercontent.com/63856055/163745736-a49d0a37-e924-4740-a55b-d2bb09f9625a.jpg)

**Reply by tejasraman** [reply]
Yes...

Try these...

```
module_blacklist=nouveau modprobe.blacklist=nouveau nouveau.runpm=1 nouveau.noaccel=1 nouveau.nofbaccel=1 nouveau.modeset=0
```

**Reply by lev666** [reply]
> Yes...
> 
> Try these...
> 
> ```
> module_blacklist=nouveau modprobe.blacklist=nouveau nouveau.runpm=1 nouveau.noaccel=1 nouveau.nofbaccel=1 nouveau.modeset=0
> ```

Did not help:(

**Comment by tejasraman** [comment]
Bad email greeting rendering, sorry
Try a Linux distro maybe?
